### ISSUE TITLE
add a "contributing" page

### ISSUE THREAD

**Issue by elstoc** [darktable-org/dtorg#170]
A common question on the pixls forums goes along the lines of "I want feature X, would you develop it if I donate some money". Rather than answer the question every time and repeat the same discussion it would be great if we could have a specific "contributing" page on darktable.org itself to which we can link. I would suggest that it should explain that the project accepts contributions of people's time (testing, bug reports, pull requests, forums etc.) but not money, giving reasons (e.g. the effort of administration/organization vs likely income, difficulty of working out how to allocate the money, tax issues etc.).

**Comment by Nilvus** [comment]
There surely a better page and more visible one to add than actual contribute one. Good idea.

**Comment by patdavid** [comment]
I agree (and am willing to write copy if/when I can find some time).

**Comment by elstoc** [comment]
I'm ok to draft something too, but IMO dtorg needs quite a bit of work in general (that was mostly waiting for the Hugo port to complete) and I think @paperdigits has some ideas, so I've held off writing any copy for now.

Perhaps worth a get-together to discuss some ideas and divvy up work?

**Comment by paperdigits** [comment]
I am going thru issues @elstoc not sure if anything ever came of this.

**Comment by paperdigits** [comment]
We sort of have that page, it is just not named very well: https://www.darktable.org/development/
